### ISSUE TITLE
JSON Schema enum dropdown not working for additionalProperties

### ISSUE THREAD

**Issue by speedakovic** [josdejong/jsoneditor#1192]
Please see this jsbin: https://jsbin.com/bezefuloga/edit?html,output

**Comment by josdejong** [comment]
Thanks for reporting and for the clear, minimal example Tomas. 

@maufl would you be interested to look into this issue? You did some very nice improvements in the `_findSchema` code recently and this is probably related (see #1161). It may be a no-brainer for you to solve. Feel free to say no 😄 .

**Comment by maufl** [comment]
Here you go: https://github.com/maufl/jsoneditor/tree/issue-1192 :) @speedakovic could you please test it with this branch?

**Comment by josdejong** [comment]
wow 🚀. Your unit test in https://github.com/maufl/jsoneditor/commit/8b8f674f04e1498b71db7911584411113eb42c39 already shows that everything works, the fix makes sense. I guess we can merge this fix if you can create a PR.

**Comment by maufl** [comment]
Yes, this fix can definitely be merged. However, I'm not sure whether this also fixes the missing dropdown in the UI and have not tested it yet (would take a bit more time than writing the unit test).

**Comment by speedakovic** [comment]
I tested https://github.com/maufl/jsoneditor/tree/issue-1192 and it seems to fix this issue, now the dropdown in the UI is shown.

**Comment by josdejong** [comment]
Fixed now in `v9.1.6`. Thanks again @maufl .
